<?xml version="1.0"?>
<launch>
    <node pkg="rap_controller" type="rap_controller.py" name="rap_controller_car1" output='screen'>
        <param name="robot_name" value="car1"/>
        <param name="role" value="leader"/>
        <param name="sim" value="false"/>
        <param name="reverse_omega" value="false"/>
        <param name="ctl_frequency" value="10"/>
    </node>
    <node pkg="rap_controller" type="rap_controller.py" name="rap_controller_car2" output='screen'>
        <param name="robot_name" value="car2"/>
        <param name="role" value="follower"/>
        <param name="sim" value="false"/>
        <param name="reverse_omega" value="false"/>
        <param name="ctl_frequency" value="10"/>
    </node>

    <node pkg="joy" type="joy_node" name="joy">
        <param name="autorepeat_rate" value="2.0" />
        <param name="deadzone" value="0.05" />
    </node>

    <node pkg="rap_controller" type="teleop_joy.py" name="teleop_joy" output="screen">
        <param name="vx_max" value="0.2"/>
        <param name="vy_max" value="0.2"/>
        <param name="wz_max" value="0.5"/>
        
        <param name="vx_axe" value="3"/>
        <param name="vy_axe" value="2"/>
        <param name="wz_axe" value="0"/>
        <param name="switch_mode_button" value="1"/> 
        <param name="speed_axe" value="5"/>
        <param name="turn_axe" value="4"/>
        
        <!--param name="vx_axe" value="4"/>
        <param name="vy_axe" value="3"/>
        <param name="wz_axe" value="0"/>
        <param name="speed_axe" value="7"/>
        <param name="turn_axe" value="6"/>
        <param name="switch_mode_button" value="0"/-->
    </node>

    <!--node pkg="rviz" type="rviz" name="rviz" args="-d $(find ros_pkg_example)/rviz/default.rviz" if="$(arg use_rviz)" /-->
</launch>
